package exceptions;

public class ThrowsCF1 {
	
	static void m1() throws InterruptedException
	{
		System.out.println("in m1");
		throw new InterruptedException();
	}
	
	static void m2() throws InterruptedException
	{
		System.out.println("in m2");
		m1();
	}
	
	static void m3() throws InterruptedException
	{
		System.out.println("in m3");
		m2();
	}

	public static void main(String[] args) {
		// TODO Auto-generated method stub
		
	System.out.println("In main");
	

	}

}
